src/Microsoft.AspNet.Razor/TagHelpers/TagHelperAttributeDescriptor.cs

```diff
@@ -116,7 +117,6 @@ public string TypeName
                 }
 
                 _typeName = value;
-                IsStringProperty = string.Equals(_typeName, typeof(string).FullName, StringComparison.Ordinal);
```

Comment:
Why move this to the ctor? By making the `IsStringProperty` public set it'll now be serialized when sent over the wire. (not needed I think).